app/views/plantings/_timeline.html.haml

```diff
@@ -0,0 +1,28 @@
+.jumbotron
+  .d-flex.justify-content-between
```

Comment:
This seems pretty dense, though I don't have good concrete suggestions on how to make it more readable for what it does - maybe some classes to help identify what regions are doing which things? Pull out things to partials/pretend they are their own sub components? 

Pre-calculate conditions up the top of the partial; to avoid putting logic in with the output HTML too much?